jcl/src/jdk.jcmd/share/classes/openj9/tools/attach/diagnostics/tools/Jps.java

```diff
@@ -71,9 +73,19 @@ public static void main(String[] args) {
 							String theCommand = vmProperties.getProperty(SUN_JAVA_COMMAND, ""); //$NON-NLS-1$
 							String parts[] = theCommand.split("\\s+", 2); /* split into at most 2 parts: command and argument string */  //$NON-NLS-1$
 							if (noPackageName) {
-								String commandName = parts[0];
-								int finalDot = commandName.lastIndexOf('.');
-								parts[0] = commandName.substring(finalDot + 1); /* if the dot is missing, we get the whole string */
+								if (parts[0].endsWith(".jar")) { //$NON-NLS-1$
```

Comment:
This should be case-insensitive (for filesystems that are likewise).

Reply:
Good idea.  Changed to `if (parts[0].toLowerCase().endsWith(".jar"))`